packages/op-coreutils/src/op_coreutils/bigquery/write.py

```diff
@@ -134,7 +138,12 @@ def overwrite_partition_static(
     )
 
 
-def _write_df_to_bq(df: pl.DataFrame, destination: str, job_config=bigquery.LoadJobConfig):
+def _write_df_to_bq(
+    df: pl.DataFrame,
+    destination: str,
+    job_config=bigquery.LoadJobConfig,
+    operation_prefix: str = "",
```

Comment:
it looks like the newly added `operation_prefix` arg is not being used. Should it be removed?